packages/components/Menu/src/MenuItemCheckbox/MenuItemCheckbox.styling.ts

```diff
@@ -43,7 +43,8 @@ export const stylingSettings: UseStylingOptions<MenuItemCheckboxProps, MenuItemC
             justifyContent: 'center',
           },
           ...(hasPressRententionForA11y && {
-            pressRetentionOffset: typeof tokens.padding === 'number' ? tokens.padding : parseFloat(tokens.padding), /// Retention of the press area outside of the checkbox equal to padding to match accessibility requirement
+            pressRetentionOffset:
+              typeof tokens.padding === 'number' ? tokens.padding : typeof tokens.padding === 'number' ? parseFloat(tokens.padding) : 0, /// Retention of the press area outside of the checkbox equal to padding to match accessibility requirement
```

Comment:
should the second typeof be `typeof tokens.padding === 'string'` ?

Reply:
Good catch